src/firefly/js/tables/ui/TableInfo.jsx

```diff
@@ -157,18 +158,18 @@ const ShowObjectTreeBtn = ({data, title}) => {
     );
 };
 
-export function KeywordBlock({sx={}, label, value, title, asLink}) {
+export function KeywordBlock({label, value, title, asLink, ...props}) {
     return (
-        <Stack direction='row' whiteSpace='nowrap' alignItems='baseline' sx={sx}>
+        <Slot component={Stack} direction='row' whiteSpace='nowrap' alignItems='baseline' slotProps={props}>
             <Keyword {...{label, value, title, asLink}}/>
-        </Stack>
+        </Slot>
     );
 }
 
-export function Keyword({label, value, title, asLink}) {
-    label = label && label + ':';
-    if (label || value) {
-        value = String(value);
+export function Keyword({label=null, value, title, asLink}) {
+    if ((label !== null) || value) {
+        label = label ? label + ':' : '\u00A0';
+        value = value ? String(value) : '\u00A0';
```

Comment:
have you tried using `'&nbsp;'` here instead? 

I tend to avoid using raw unicode characters where possible for readability.

Reply:
'&nbsp;' doesn't work.  It treats it as plain text.